api-docs/api-reference/methods/users/post-forgot-pwd-v2.0.rst

```diff
@@ -0,0 +1,74 @@
+.. _post-forgot-pwd-v2.0:
+
+Forgot password
+~~~~~~~~~~~~~~~
+
+.. code::
+
+    POST /v2.0/users/RAX-AUTH/forgot-pwd
+
+Use the Forgot password operation to get a password reset token delivered to
+the email address associated with your account.
+
+After you receive the password reset token, use the new token to submit a
+:ref:`Reset password <post-reset-pwd-v2.0>` request. You can only use this
+token to reset your password.
+
+.. note::
+   The reset token has a limited lifespan. If the reset operation fails, submit
+   another forgot password request to generate a new token.
+
+
+This table shows the possible response headers for this operation:
+
+.. csv-table::
+   :header: Name, Type, Description
+   :widths: 2, 2, 2
+
+   X-User-Name, Header, Value for name provided on operation
+
+This table shows the possible response codes for this operation:
+
+.. csv-table::
+   :header: Response Code, Name, Description
+   :widths: 2, 2, 2
+
+   204, No Content, No Content
+
+Request
+-------
+
+This table shows the body parameters for the request:
+
+.. csv-table::
+   :header: Name, Type, Description
+   :widths: 2, 2, 2
+
+   RAX-AUTH:forgotPasswordCredentials, Object, Provides username and portal for forgot password.
+   RAX-AUTH:forgotPasswordCredentials.**username**, String *(Required)*, The user name ofr the Rackspace Cloud account.
+   RAX-AUTH:forgotPasswordCredentials.**portal**, String *(Required)*,"Authentication portal. Specify one of the following:
+
+   - ``cloud_control``"
+
+**Example: Forgot password: XML request**
+
+.. code::
+
+   <?xml version="1.0" encoding="UTF-8"?>
+   <forgotPasswordCredentials portal="LOGIN" username="billybob"
+        xmlns="http://docs.rackspace.com/identity/api/ext/RAX-AUTH/v1.0"
+        xmlns:OS-KSADM="http://docs.openstack.org/identity/api/ext/OS-KSADM/v1.0"
+        xmlns:OS-KSCATALOG="http://docs.openstack.org/identity/api/ext/OS-KSCATALOG/v1.0"
+        xmlns:atom="http://www.w3.org/2005/Atom"
+        xmlns:identity="http://docs.openstack.org/identity/api/v2.0"/>
+
+**Example: Forgot password: JSON request**
+
+.. code::
+
+   {
+     "RAX-AUTH:forgotPasswordCredentials": {
+       "username": "billybob",
+       "portal": "LOGIN"
+     }
+   }
```

Comment:
Add HTTP headers to examples as shown here: https://developer.rackspace.com/docs/cloud-identity/v2/api-reference/token-operations/#authenticate-with-multi-factor-authentication-credentials-request.

Also, since the response has a header, add a Response section.  Move the table with the response header below that heading, and provide an example that shows the http header and response codes.